spec/requests/contributors_spec.rb

```diff
@@ -38,6 +38,19 @@
     end
   end
 
+  describe 'GET /conversations' do
+    it 'returns the conversation of the contributor' do
+      received_message = create(:message, text: 'the received message', recipient: contributor,
+                                          request: the_request)
+      sent_message = create(:message, text: 'the sent message', request: the_request, sender: contributor)
```

Comment:
let is a memoized helper method that is only executed when called in the spec itself or in an before block. Here we need a setup to be run before the assertion. So why I decided to do it this way:
1. No need for let as it is not reused in another spec. When using let we would additionally need a before block to call those methods or directly call them in the spec nevertheless
2. In the before block alone you can't define variables accessible in the spec.
3. This implementation is IMHO the most telling (create sth first) then assert against without generating any overhead

Reply:
let is a memoized helper method that is only executed when called in the spec itself or in an before block. Here we need a setup to be run before the assertion. So why I decided to do it this way:
1. No need for let as it is not reused in another spec. When using let we would additionally need a before block to call those methods or directly call them in the spec nevertheless
2. In the before block alone you can't define variables accessible in the spec.
3. This implementation is IMHO the most telling (create sth first) then assert against without generating any overhead

Reply:
I typically use `let!` if I want it to be created instead of using a before block to create it or in the spec itself. That being said, the only reason I commented is because I have seen that we have this pattern, but I am perfectly ok with different styles. Typically, I like to let a style guide decide, so I was curious what rubocop-rspec had to say about this. 

I learned that this is an open discussion with arguments against include [thoughtbot's article Let's Not](https://thoughtbot.com/blog/lets-not)

I'll think about my own usage of `let` in the future.